# Frequently Asked Questions

!!! Question "What is GeneWeaver? What happened to “The Ontological Discovery Environment”?"
    A. The Ontological Discovery Environment was conceived of as a tool for the integration of biological functions based on
    the molecular processes that subserved them. From these data, an empirically derived ontology may one day be inferred.
    Sounds like a mouthful? We think so, too. Moreover, our acronym, ODE, sounds like “ordinary differential equations”,
    “open development environment”, “Ohio Department of Education”, and the airport in Odense, Denmark. Our users have found
    the system valuable for a wide range of applications in the arena of functional genomic data integration. While the
    underlying algorithms of The Ontological Discovery Environment can be extended to many contexts, we chose to rename the
    system “GeneWeaver” to reflect the emphasis on genes and genomes, allowing our users to weave together the many complex
    relations among processes, pathways and functions implicit in functional genomics experiments.

!!! Question "How is GeneWeaver different from gene set enrichment or ontology over-representation tools?"
    There are many statistical tools for the analysis of gene set overrepresentation, and it is indeed possible to perform
    similar analyses using some of the functions in GeneWeaver. However, GeneWeaver’s primary focus and strength is in using
    gene sets to organize biological functions. GeneWeaver enables highly flexible set-set comparisons of both user
    submitted and curated gene sets. The suite of combinatorial tools enable large collections of user submitted tools to be
    compared to each other, and the hierarchical similarity tools enable classification and organization of gene sets based
    on the genes they contain. This allows discovery of hidden relations among common biological processes, even if those
    processes have been studied using highly diverse species, analytic methods and approaches. The GeneWeaver tools provide
    facile data integration and harmonization, and enable user directed integration of new and published results. Major
    incorporated data from other resources provides a wealth of other sources of contextual information which facilitate
    interpretation of these discoveries.

!!! Question "How do I add my own gene sets to GeneWeaver?"
    There are step-by-step instructions available in the Uploading Gene Sets section.

!!! Question "How do I add Open Biological Ontology annotation to my gene set?"
    Browse to your GeneSet and click the “edit” link. Scroll to the bottom of the page and use the Tree Browser to select
    entries for your GeneSet. To change the OBO source, use the drop box at the top of the tree display. Finally, to remove
    any extraneous entries, you can use the little red ‘x’ on the left side. After saving the changes, your new information
    will be displayed and your GeneSet will be searchable using any of the ontologies selected.

!!! Question "How do I change the abbreviation, name etc. for my gene set?"
    Go to the View My GeneSets page, the link is found in the Manage Gene Sets column. Scroll to your GeneSet and click
    the “edit” link. Then simply change the values and save your changes. The new text will be displayed immediately.

!!! Question "I set my threshold too high/low. How do I change it?"
    Go to the View My GeneSets page (the link is found in the Manage GeneSets drop-down on the navigation bar). Scroll to
    your GeneSet and click the link on the gene set name in order to see the GeneSet Information page. Click the Set
    Threshold button. Then simply fix the thresholds and save the changes. The new thresholds will be applied immediately.

!!! Question "I’ve got great results, but how do I make a high resolution image for my presentation?"
    Each tool has a link to export the result as a PDF. Save the file and open it in Adobe Acrobat, Inkscape or other
    software. Save as PNG. This PNG file can be easily inserted into MS Powerpoint presentations or Word documents.

!!! Question "I uploaded a file with 200 genes, but it says that my gene set is empty?"
    If there was no error reported, you probably set your threshold too high/low, see the previous question. If there was
    an error, your data probably uses a different microarray or gene ID type than what was provided on the upload page.

!!! Question "A public gene set is improperly labeled. How do I report this?"
    From the GeneSet’s information page, click the bug icon on the top navigation bar and let us know what specifically
    needs updating. Include the “GS” number.

!!! Question "How are homologous genes identified?"
    We use homologene along with any information provided by the reference genome. ex: RGD provides MGI ids as well.

!!! Question "My gene sets are listed as ‘deprecated’. What does this mean?"
    If a newer version of a Gene Set in one of your projects is available, the version you stored is marked “deprecated.”
    Clicking on the provided icon will update your project with the latest version of this data. New versions are available
    when we update data from external sources, e.g. MP and GO annotations, or when the GeneSet Metadata has been updated.

!!! Question "How should I cite GeneWeaver in my research?"
    Please cite: 
    > Erich J. Baker, Jeremy J. Jay, Jason A. Bubier, Michael A. Langston, and Elissa J. Chesler. GeneWeaver:
    a web-based system for integrative functional genomics. Nucl. Acids Res. (2012) 40(D1): D1067-D1076. Please visit other
    relevant publications.

!!! Question "What do all the acronyms on the site stand for?"
    - DRG (Drug-Related Genes), 
    - CTD (Comparative Toxicogenomics Database), 
    - MP (Mammalian Phenotype Ontology),
    - HP (Human Phenotype Ontology), 
    - ABA (Allen Brain Atlas), 
    - GO (Gene Ontology), 
    - MeSH (Medical Subject Headings).
